### DISCUSSION TITLE
predefined access code vs. separate override password?

### DISCUSSION THREAD

**Discussion by Jonathan-B-Reichel** [proginosko/LeechBlockNG#411] [Q&A] [answered]
Hi, would someone be able to explain the difference between the predefined access code and the separate override password options in the Temporary Override administration? It looks from the documentation that it's intended to be self-explanatory or obvious, but to me they book look like they'd be the same thing

**Comment by proginosko** [comment, marked as answer]
The predefined access code works like the random access code, except that you specify it yourself: you see it in the prompt and have to copy it into the text field. So you would want to make it something long and laborious to type out.

The override password works just like any other password: you need to know it beforehand in order to activate the override. LeechBlock allows for a separate override password (i.e., separate from the password used to limit access to the options) because some users want to delegate the decision to override to someone else (e.g., friend or relative).

Does that make sense?

**Reply by slepto** [reply]
Yeah, that's pretty clear, thanks.  

It's pretty useful to have flexibility in setting the override activation mechanism.  It would extend the flexibility to enable multiple mechanisms, rather than picking one.  For example, use the predefined access code for self-initiated blocks of a limited duration, but have the option to also delegate longer overrides to a friend or relative.  Would the "idea" section be the right place to register this as a request?  I also notice that the tapatalk forum is still active.

**Reply by Jonathan-B-Reichel** [reply]
Agreed on both points; thanks!

**Reply by proginosko** [reply]
You were right that the documentation for these options sucked. 🙂 I've fixed that now.
